compiler/damlc/daml-compiler/src/DA/Daml/Compiler/Repl.hs

```diff
@@ -168,7 +175,21 @@ runRepl opts mainDar replClient ideState = do
                 hPutStrLn stderr ("Package could not be loaded: " <> show err)
                 exitFailure
             Right _ -> pure ()
-    go moduleNames 0 []
+    let initReplState = ReplState
+          { moduleNames = moduleNames
+          , bindings = []
+          , lineNumber = 0
+          }
+    -- TODO[AH] Use Repl.evalReplOpts once we're using repline >= 0.2.2
```

Comment:
Can we upgrade already?

Reply:
Sadly not, there's a version constraint on `haskeline` and that's a core package. So, this is blocked on a GHC upgrade.

Reply:
Ah right :disappointed: